javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -31,11 +31,17 @@ private module MySql {
   /** Gets the package name `mysql` or `mysql2`. */
   API::Node mysql() { result = API::moduleImport(["mysql", "mysql2"]) }
 
+  /** Gets a reference to `mysql.createConnection`. */
+  API::Node createConnectionCallee() { result = mysql().getMember("createConnection") }
+
   /** Gets a call to `mysql.createConnection`. */
-  API::Node createConnection() { result = mysql().getMember("createConnection").getReturn() }
+  API::Node createConnection() { result = createConnectionCallee().getReturn() }
```

Comment:
Again, not a fan of the naming. I think the name `createConnection` is a better fit for what is currently called `createConnectionCallee`. This predicate is now only used once, so we might as well inline it.